debian-8/init_container.sh

```diff
@@ -20,11 +20,55 @@ sed -i "s/SSH_PORT/$SSH_PORT/g" /etc/ssh/sshd_config
 /usr/sbin/sshd
 
 appPath="/home/site/wwwroot/"
+userStartupCommand="$@"
+
+function tryStripWwwRootPathFromInput() {
+    local originalArg="$1"
+    # Strip out the beginning of the argument if it starts with the the appPath
+    # Examples: /home/site/wwwroot/app.dll => app.dll
+    #           /home/site/wwwroot/app => app
+    if case $originalArg in $appPath*) true;; *) false;; esac; then
+        sanitizedArg="${originalArg/$appPath/}"
+        updatedPath="true"
+    else
+        sanitizedArg="$originalArg"
+    fi
+}
+
+# When run from copy is enabled, Oryx tries to run the app from a different directory (local to the container),
+# so sanitize any input arguments which still reference the wwwroot path. This is true for VS Publish scenarios.
+# Even though VS Publish team might fix this on their end, end users might not have upgraded their extension, so
+# this code needs to be present.
+if [ "$APP_SVC_RUN_FROM_COPY" = true ]; then
+    if [ $# -ne 0 ]; then
```

Comment:
how about `userStartupCommand=$(echo $userStartupCommand | sed 's!/home/site/wwwroot/!!g')`

Reply:
I like it!...although the current code only checks for beginning of an argument, I think it would be uncommon to use `/home/site/wwwroot` to be used in middle of a string..so i will update to the `sed` way...thanks